src/unix/process.c

```diff
@@ -220,10 +234,11 @@ static void uv__write_errno(int error_fd) {
  * avoided. Since this isn't called on those targets, the function
  * doesn't even need to be defined for them.
  */
-static void uv__process_child_init(const uv_process_options_t* options,
-                                   int stdio_count,
-                                   int (*pipes)[2],
-                                   int error_fd) {
+static void uv__process_child_init(
+    const uv_process_options_t* options,
+    int stdio_count,
+    int (*pipes)[2],
+    int error_fd) {
```

Comment:
Still I don't understand the style code change here. It was ok before and if I see things correctly the code hasn't really changed.

Reply:
I wanted to keep consistency within the file, since several of the new calls here now must be wrapped this way to fit the line-width constraint. This is how it is specified to wrap them in the libuv style guide: https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions